x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/api/install_prebuilt_rules_and_timelines/install_prebuilt_rules_package.ts

```diff
@@ -19,24 +22,45 @@ export async function installPrebuiltRulesPackage(
   config: ConfigType,
   context: SecuritySolutionApiRequestHandlerContext
 ) {
-  // Get package version from the config
+  const pkgVersion = await findLatestPackageVersion(config, context, PREBUILT_RULES_PACKAGE_NAME);
+
+  return context
+    .getInternalFleetServices()
+    .packages.ensureInstalledPackage({ pkgName: PREBUILT_RULES_PACKAGE_NAME, pkgVersion });
```

Comment:
Under the hood, this endpoint still uses this Fleet method right? Just trying to understand if there's any performance gain here: you explained that some stuff that Fleet does on their side -like checking references- is not necessary for installing our pacakge. But I guess that's an improvement not for this PR, but when we get content-only packages?

Reply:
Nothing changes regarding the package installation method, that's right. We will implement optimizations in package installation separately. The only performance gain in this PR is that we do not invalidate prebuilt rules when the package is already installed.

Previously:
`Fetch rules initially -> Upgrade rules package -(always)-> Re-fetch rules`

Now:
`Fetch rules initially -> Upgrade rules package -(only if there's a new package version)-> Re-fetch rules`

This will result in fewer redundant API requests from Kibana.